### PR TITLE
Create Makefile

### PR DESCRIPTION
**Pull request by misobarisic**
The title is self-explanatory.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+all:
+	@echo Run \'make install\' to install notflix.
+
+install:
+	@mkdir -p $(DESTDIR)$(PREFIX)/bin
+	@cp -p notflix $(DESTDIR)$(PREFIX)/bin/notflix
+	@chmod 755 $(DESTDIR)$(PREFIX)/bin/notflix
+
+uninstall:
+	@rm -rf $(DESTDIR)$(PREFIX)/bin/notflix
```
Comment (alpheratz0) [inline Makefile:12]: Why use recursive flag if the target is not a directory? I would go with:

```make
uninstall:
	@rm -f $(DESTDIR)$(PREFIX)/bin/notflix
```

Comment (misobarisic) [reply Makefile:12]: Oh whoops. Force of habit. My bad 😅

**Comment by Si-Ki** [comment]
no offense.... but don't you think that's just bloat, considering what we are going to install with that make file?

**Comment by misobarisic** [comment]
> no offense.... but don't you think that's just bloat, considering what we are going to install with that make file?

None taken. It is bloat to an extent. 258 bytes to be precise.

Other than that, I don't see a reason as to why it should not be implemented. Other projects use it too. Q

**Comment by misobarisic** [comment]
> no offense.... but don't you think that's just bloat, considering what we are going to install with that make file?

None taken. It is bloat to an extent. 258 bytes to be precise.

Other than that, I don't see a reason as to why it should not be implemented. Other projects use it too.

**Comment by Si-Ki** [comment]
No, I meant bloat as in... like unnecessary feature... like we don't even need to worry about sizes these days, again no offense, as I see it we are just two people with different pov communicating. Also, I see how the makefile makes sense as the project grows... but as now it's just one extra file that's going to just be there even after the user uninstalls notflix using it... besides it's not something to dwell on so, 🤝 and did you check out my pull request, do lemme know what you think about it. Peace ✌️

**Pull request closed without merging**